cluster/gce/gci/configure-helper.sh

```diff
@@ -77,12 +77,21 @@ function setup-logrotate() {
     compress
     maxsize 10M
```

Comment:
Nevermind. I recalled some discussion on google-containers group recommending to redirect logs to files.
So, is 10Mb here fine?